### PR TITLE
Improve help for !show and /show

### PR DESCRIPTION
**Pull request by sirKiraUzumaki**
Changed help of "/show" to "!show" and added help for "/show".

### CODE REVIEW

File: server/chat-commands/info.ts [hunk 1]
```diff
@@ -2650,7 +2650,10 @@ export const commands: ChatCommands = {
 		this.runBroadcast();
 		this.sendReplyBox(buf);
 	},
-	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
+	showhelp: [
+		`/show [url] - shows you an image or video url in chat.`,
+		`!show [url - shows an image or video url in chat. Requires: whitelist % @ # &]`,
```
Comment (KrisXV) [inline server/chat-commands/info.ts:2655]:
```suggestion
		`!show [url - shows an image or video url in chat. Requires: whitelist % @ # &`,
```

Comment (KrisXV) [inline server/chat-commands/info.ts:2655]:
```suggestion
		`/show [url] - shows you an image or video url in chat.`,
		`!show [url - shows an image or video url in chat. Requires: whitelist % @ # &`,
```

**Comment by AnnikaCodes** [comment]
Why does the broadcasted version need its own help message?

**Comment by sirKiraUzumaki** [comment]
> Why does the broadcasted version need its own help message?

Most of the commands have help for both broadcasted and non broadcasted version.
like !code [code] - Broadcasts code to a room. Accepts multi-line arguments. Requires: + % @ & #
/code [code] - Shows you code. Accepts multi-line arguments.

File: server/chat-commands/info.ts [hunk 2]
```diff
@@ -2650,7 +2650,10 @@ export const commands: ChatCommands = {
 		this.runBroadcast();
 		this.sendReplyBox(buf);
 	},
-	showhelp: [`/show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`],
+	showhelp: [
+		`/show [url] - shows you an image or video url in chat.`,
+		`!show [url] - shows an image or video url in chat. Requires: whitelist % @ # &`,
```
Comment (mia-pi-git) [inline server/chat-commands/info.ts:2655]:
```suggestion
		`/show [url] - Shows you an image or video url in chat.`,
		`!show [url] - Shows an image or video url in chat. Requires: whitelist % @ # &`,
```

Comment (Zarel) [reply server/chat-commands/info.ts:2655]:
```suggestion
		`/show [url] - Shows you an image or YouTube video.`,
		`!show [url] - Shows an image or YouTube to everyone in a chatroom. Requires: whitelist % @ # &`,
```

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**